backend/src/main/java/com/google/slurp/servlets/ServerListener.java

```diff
@@ -0,0 +1,46 @@
+package com.google.slurp.servlets;
+
+import java.io.IOException;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+
```

Comment:
Remove space.

Reply:
Ah I didn't realize that the Google Java style guide disallowed spaces unless they're between static and non-static imports. This has been fixed.